app/views/components/inputDate.scala.html

```diff
@@ -46,21 +46,21 @@ <h1 class="govuk-fieldset__heading" id ="page-heading">
   items = Seq(
     InputItem(
       id = s"$id.day",
-      classes = s"govuk-input--width-2${if(form(s"$id.day").hasErrors || form.errors.exists(_.args.contains("day")) || form.errors.exists(_.args.size > 1)) " govuk-input--error" else ""}",
+      classes = s"govuk-input--width-2 ${if(form(s"$id.day").hasErrors || form.errors.exists(_.args.contains("day")) || form.errors.exists(_.args.contains("diwrnod"))) "govuk-input--error" else ""}",
```

Comment:
To the reviewer, thinking of changing this to be form.errors.exists(_.args.contains(messages("date.day")) so that its more robust in the future?

Reply:
Not sure if I'm implementing wrong on my end, but it seems like the above suggestion makes it highlight only one field again?